src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java

```diff
@@ -223,6 +223,7 @@ protected void clickByListElementText(List<WebElement> list, String text) {
     throw new RuntimeException(getNoTextInListErrorMessage(text));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00112)